Kitodo/src/main/java/de/sub/goobi/forms/MassImportForm.java

```diff
@@ -682,7 +686,7 @@ public void setUsablePluginsForFiles(List<String> usablePluginsForFiles) {
      * @return boolean
      */
     public boolean getHasNextPage() {
-        java.lang.reflect.Method method;
+        /*Method method;
```

Comment:
Why did you comment this method out? At least add a TODO comment why this code is commented out.